src/docs/software/using/R.md

```diff
@@ -428,7 +428,7 @@ those on GitHub.
 Installing `devtools` is somewhat memory-intensive and has several
 dependencies. The following example shows how to run an interactive session
 with 4 CPUs, load the modules for the necessary dependencies, and install
-`devtools` for R version 4.2.0.
+`devtools` for R version 4.2.0. Note: these dependencies will also work for installing the popular library `tidyverse`.
```

Comment:
We probably should update the R version listed here as well, to match the example below?